op-challenger/config/config.go

```diff
@@ -24,9 +25,11 @@ var (
 	ErrMissingCannonSnapshotFreq     = errors.New("missing cannon snapshot freq")
 	ErrMissingCannonRollupConfig     = errors.New("missing cannon network or rollup config path")
 	ErrMissingCannonL2Genesis        = errors.New("missing cannon network or l2 genesis path")
-	ErrCannonNetworkAndRollupConfig  = errors.New("only specify one of network or rollup config path")
-	ErrCannonNetworkAndL2Genesis     = errors.New("only specify one of network or l2 genesis path")
-	ErrCannonNetworkUnknown          = errors.New("unknown cannon network")
+	ErrCannonNetworkAndRollupConfig  = errors.New(
```

Comment:
Shouldn't be re-formatted.

Reply:
Sorry about this - my editor keeps messing up. Will fix this so I don't accidentally reformat files going forward.